packages/dart/sshnp_flutter/pubspec.yaml

```diff
@@ -31,12 +31,12 @@ dependencies:
   intl: any
   macos_ui: ^2.0.0
   mocktail: ^1.0.0
-  noports_core: ^5.0.0
+  noports_core: ^6.0.0
```

Comment:
noports_core: ^6.0.1

Reply:
@CurtlyCritchlow still need to change the noports_core dependency